packages/core/src/components/FilterGroup/RightPanel/RightPanel.tsx

```diff
@@ -49,8 +49,13 @@ export const HvFilterGroupRightPanel = ({
   } = useContext(HvFilterGroupContext);
 
   const activeGroupOptions = useMemo(
-    () => filterOptions[activeGroup]?.data.map((option) => option.id) || [],
-    [filterOptions, activeGroup]
+    () =>
+      filterOptions[activeGroup]?.data
```

Comment:
I just noticed another thing when selecting the "All" when there's a filter: 

https://github.com/lumada-design/hv-uikit-react/assets/43220251/9d89ef67-7ecb-42ec-b2f4-35e0fe4259ca

I think we need to update `handleSelectAll`.

Reply:
@MEsteves22 yeah, that's a weird behavior. I updated the PR, let me know what you think.